service1/handler.py

```diff
@@ -1,7 +1,7 @@
 import requests
 
 def handle(st):
-    r = requests.post("http://gateway:8080/async-function/fn-b")
+    r = requests.post("http://128.110.153.209:8080/async-function/fn-b")
```

Comment:
You shouldn't be putting a hard-coded IP in here.

Reply:
Sorry for that, just wanted to make the setup ready to show it to you.